public/js/draw/draw.js

```diff
@@ -389,3 +389,99 @@ function regionClicked(e) {
         prevY = position.y;
     }
 }
+
+
+/**
+ * Convert JSON data into SVG elements then populate the canvas with them.
+ */
+function jsonToSvg(jsonStr) {
+    'use strict';
+
+    var jsonData = JSON.parse(jsonStr);
```

Comment:
@david-yz-liu I noticed that arrows have been introduced in #805. Do we have plans to move to `let` over `var`?